### ISSUE TITLE
Empty BitString decoded incorrectly?

### ISSUE THREAD

**Issue by joernheissler** [wbond/asn1crypto#102]
```python
from asn1crypto.core import BitString

bits = BitString(())
buf = bits.dump(True)
print(buf.hex())

bits2 = BitString.load(buf)
print(bits2.native)
print(bits2.dump(True).hex())
```

```
030100
(0, 0, 0, 0, 0, 0, 0, 0)
03020000
```

I believe that the correct encoding for the empty bit string is indeed `03 01 00`. When decoding, the result should be the empty bitstring again.

> If the bitstring is empty, there shall be no subsequent octets, and the initial octet shall be zero.

**Comment by wbond** [comment]
Is this related to #97?

**Comment by wbond** [comment]
This is fixed by e57ad1815abde7f3bb6b0da30abfda2b629fb9b7
